src/edfi-paging-test/edfi_paging_test/api/request_client.py

```diff
@@ -72,9 +73,30 @@ def _build_query_params_for_page(self, page_index, page_size: int) -> str:
         page_offset = (page_index - 1) * page_size
         return f"offset={page_offset}&limit={page_size}"
 
+    def _get_version(self) -> VersionInfo:
+        """Send an HTTP GET request for the api version endpoint.
+
+        Returns
+        -------
+        VersionInfo
+        """
+        if self.version_info is None:
+            logger.info("Getting api version information.")
```

Comment:
Suggest DEBUG and a message that is more clear on the overall intent: getting API metadata.

Reply:
yes it is not version info we need here but "getting API metadata" sounds like the open api metadata that we will be calling later to pull resource endpoints.